src/AlertBar/Icon.js

```diff
@@ -1,6 +1,8 @@
-import React from 'react'
 import propTypes from 'prop-types'
-import { Valid, Warning, Error, Info } from '../icons/Status'
+import React from 'react'
+
+import { statusPropType } from '../common-prop-types'
```

Comment:
General comment: Other imports use the `.js` suffix.

Reply:
Seems to be this file? I think there shouldn't be any `.js`

Reply:
I see ... `common-prop-types` is a directory.